app/src/main/kotlin/co/netguru/baby/monitor/client/feature/voiceAnalysis/RecordingController.kt

```diff
@@ -26,8 +26,8 @@ class RecordingController @Inject constructor(
             .map {
                 val data = when {
                     isMachineLearningWithEnoughData(it) -> RecordingData.MachineLearning(
-                        it.first.toByteArray(),
-                        it.second.toShortArray()
+                        it.first.takeLast(MachineLearning.DATA_SIZE * 2).toByteArray(),
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/c8073f95e65efde091cb24d7d4757647fb4cf9af/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/voiceAnalysis/RecordingController.kt#L29"></span>This expression contains a magic number. Consider defining it to a well named constant.</td>
    </tr>
  </tbody>
</table>